FreeRTOS/FreeRTOS
# Apply compile option change on RP2040

Apply compile option change on RP2040

Description
-----------

As discussion in [FreeRTOS-Kernel issue#731](https://github.com/FreeRTOS/FreeRTOS-Kernel/issues/731), adding compile options on RP2040 to check compile result by CI.

Test Steps
-----------

Follow steps on CI to compile 4 different RP2040 images successfully.
```
cmake -B build -DPICO_SDK_PATH=pico-sdk -GNinja
ninja -C build --verbose
```

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
